creating-pull-requests.md

```diff
@@ -0,0 +1,131 @@
+# Creating a Pull Request
+
+This page goes over the steps for creating a Pull Request (PR), including some dos and don’ts.
+
+Note: the terms _Pull Request_ and _Merge Request_ can be used interchangeably. The same is true for the terms _issue_, _ticket_, and _stories_.
+
+## Before you start working
+
+Make sure that there is an issue created and assigned to you. Work on a feature branch that is named accordingly. It might also be a good idea to validate the approach you intend to take in case it is a big or complex issue.
```

Comment:
For clarity an example of branch names is a good idea. Which one would be correct?
Most tickets have names like: ```CN - EPK/Post Unveil - Shoe carousel update copy Morpho```

Reply:
If possible it should contain the number of the ticket and a descriptive (human readable)  name

Reply:
Including some good and bad examples might be a good addition.

Reply:
I'm not sure what to put here. Suggestions for good/bad?

Reply:
If possible we could write the PRs similar to how we write commits. eg: 
Title: #373 || Technical - Frontend(_if the codebase is shared_) - Do something(_use imperative form_)

Description: Try to give some background if necessary and explain why you are doing it.

In the end practice makes perfect and by reviewing each others merges/commits/PRs we'll be able to have a more concise and clear structure.

Reply:
can we close/merge this an reopen a new issues to improve it later?

Reply:
Opened an issue: https://github.com/mediamonks/frontend-coding-standards/issues/58